File: csharp/ButtplugCSharpFFI/ButtplugDeviceException.cs
using System;

namespace Buttplug
{
    public class ButtplugDeviceException : ButtplugException
    {
        /// <inheritdoc />
        public ButtplugDeviceException()
        {
        }

        /// <inheritdoc />
        public ButtplugDeviceException(string aMessage) : base(aMessage)
        {
        }

        /// <inheritdoc />
        public ButtplugDeviceException(string aMessage, Exception aInner) : base(aMessage, aInner)
        {
        }
    }
}

File: java/lib/src/main/java/io/buttplug/exceptions/ButtplugDeviceException.java
package io.buttplug.exceptions;

public class ButtplugDeviceException extends ButtplugException {
    public ButtplugDeviceException(String msg) {
        super(msg);
    }

    public ButtplugDeviceException(String msg, Exception inner) {
        super(msg, inner);
    }
}
